src/snowflake/connector/connection.py

```diff
@@ -829,12 +829,10 @@ def cmd_query(self,
                 is_file_transfer
             )
 
-        url_parameters = {REQUEST_ID: request_id}
-
         ret = self.rest.request(
-            '/queries/v1/query-request?' + urlencode(url_parameters),
+            '/queries/v1/query-request?',
```

Comment:
If we leave the `?` at the end we end up with urls like `{host}/queries/v1/query-request?&request_id=asd&request_guid=def`. Let's remove the extra `?`.
```suggestion
            '/queries/v1/query-request',
```